sycl/test-e2e/Matrix/SG32/joint_matrix_annotated_ptr.cpp

```diff
@@ -16,7 +16,7 @@
 // RUN: %{run} %t.out
 
 // Currently row major B fails when annotated_ptr is used
-// XFAIL: gpu
+// XFAIL: !(linux && arch-intel_gpu_bmg_g21)
```

Comment:
to keep it passing on CPU
```suggestion
// XFAIL: gpu && !(linux && arch-intel_gpu_bmg_g21)
```